manifests/course/practitioner.pp

```diff
@@ -22,4 +22,6 @@
   class { 'classroom::facts':
     coursename => 'practitioner',
   }
+
+  Yumrepo <| |> -> Package <| |>
```

Comment:
are you certain there's nothing weird here? e.g. a yumrepo installed via package?